learner_utils.py

```diff
@@ -28,6 +28,22 @@ def learner_decisions(subpops, current = None, min_fn=quadratic_min):
             new_thetas.append(theta_i)
     return new_thetas
 
+def learner_max_price(thetas, prices, subpops, min_fn=quadratic_min):
+    n_learners = len(thetas)
+    new_prices = [0] * len(prices)
+    #price_candidates = np.linspace(0, 100, 51)
+    for i in range(n_learners):
+        price_candidates = [prices[i] + 1, prices[i] * 2, prices[i] / 2]
+        profit_candidates = []
+        for p in price_candidates:
+            tmp = prices[i]
+            prices[i] = p
+            new_alphas = subpop_decisions(thetas, subpops, epsilon=0.1, prices=prices, dryrun=True)
+            profit_candidates.append(new_alphas[:, i].sum() * p)  # warning: assumes beta is uniform
```

Comment:
will fix this calculation to account for non-uniform beta

Reply:
maybe make a comment in the code 
#TODO